cache/redis_internal_test.go

```diff
@@ -45,7 +45,29 @@ func TestNewRedis(t *testing.T) {
 	c := v.(*redisCache)
 
 	assert.NoError(t, err)
-	assert.Equal(t, 12, c.client.Options().PoolSize)
+	clusterClient, isCluster := c.client.(*redis.ClusterClient)
+	if isCluster {
+		assert.Equal(t, 12, clusterClient.Options().PoolSize)
```

Comment:
This is an impossible outcome to the test.

Reply:
OK

Reply:
yeah